src/containers/contactsPage/ContactsPage.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+
+import { ContactForm } from "../../components/contactForm/ContactForm";
+import { TileList } from "../../components/tileList/TileList";
+
+export const ContactsPage = ({ contacts, addContact }) => {
+  const [name, setName] = useState("");
+  const [phone, setPhone] = useState("");
+  const [email, setEmail] = useState("");
+  const [duplicate, setDuplicate] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    e.target[0].value = "";
```

Comment:
This is an anti-pattern. We shouldn't be resetting the value of the inputs like this.

We will need to bind the input's values to the value. So in `<ContactForm />`

```tsx

<input value={name} ... />
```

Reply:
Done